armi/reactor/reactors.py

```diff
@@ -2284,18 +2293,11 @@ def processLoading(self, cs, dbLoad: bool = False):
                     if a.hasFlags(nonUniformAssems, exact=True):
                         continue
                     a.makeAxialSnapList(self.refAssem)
-
             if not cs["inputHeightsConsideredHot"]:
                 runLog.header(
-                    "=========== Axially expanding all (except control) assemblies from Tinput to Thot ==========="
+                    "=========== Axially expanding all assemblies (except control) from Tinput to Thot ==========="
```

Comment:
Maybe change this to "Thermal Axial Expansion of Core Assemblies" as a header and then when you get to control just create a runLog info that CONTROL is being skipped. I'm curious if there should be a setting to say which assemblies to skip and we can just configure our application to neglect control rods.

Reply:
^^^ Yes, this!

Reply:
One thing with changing from "all" to "Core". In this case, we are thermally expanding _all_ assemblies - both Core and blueprints assemblies. So it may be confusing if we say just Core when we are doing more than just the Core assems. 

@jakehader either way, your idea feels worthy of a separate PR to me. Maybe create an issue for it and we can reference it in a new PR?

Reply:
Sure, the configuration can be another PR. Just wanted to give that feedback

Reply:
Issue made. https://github.com/terrapower/armi/issues/832